server/test/controllers/ControllerTest.scala

```diff
@@ -21,7 +23,9 @@ class ControllerTest extends PlaySpec with MockFactory {
     "post fail with scalaTestRunner when addTask" should {
       "result BadRequest with error" in {
```

Comment:
would be nice to split all out cases into 3 sections: given, when, then. It speeds up the test reading in times